<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loginn</class>
 <widget class="QDialog" name="Loginn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sign In</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/icon/C:/Users/PC-QUEREN/Downloads/connexion (2).png</normaloff>:/icon/C:/Users/PC-QUEREN/Downloads/connexion (2).png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vous n'avez pas encore de compte?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connectBtn">
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>11</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
border : none ;
color:rgb(84, 95, 255);</string>
      </property>
      <property name="text">
       <string>S'inscrire</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>271</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; 
border-radius: 15px;
padding:5px;
</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(149, 172, 255);
border: 1px solid blue; 
border-radius: 15px;
padding:5px;
</string>
      </property>
      <property name="text">
       <string>Se connecter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica CE 35 Thin</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(99, 159, 255);</string>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLineEdit" name="usernameConEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: transparent; 
    border: none; 
    border-bottom: 1px solid black; 
    color: black;  
    padding: 5px; 
    font-size: 14px;                  
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mot de passe : </string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="passwordConEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: transparent; 
    border: none; 
    border-bottom: 1px solid black; 
    color: black;  
    padding: 5px; 
    font-size: 14px;                  
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom de l'utilisateur :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="usernameError">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="passwordError">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
